optimism/deploy/002_exchangeIssuanceZeroEx.ts

```diff
@@ -0,0 +1,72 @@
+import "module-alias/register";
+
+import { HardhatRuntimeEnvironment as HRE } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+
+import {
+  prepareDeployment,
+  findDependency,
+  getContractAddress,
+  getCurrentStage,
+  saveContractDeployment,
+  stageAlreadyFinished,
+  trackFinishedStage
+} from "@utils/index";
+
+import { DEPENDENCY } from "../deployments/utils/dependencies";
+import { Address } from "@utils/types";
+import { CONTRACT_NAMES } from "../deployments/constants/002_exchangeIssuanceZeroEx";
+
+const {
+  WETH,
+  CONTROLLER,
+  ZERO_EX_EXCHANGE
+} = DEPENDENCY;
+
+const CURRENT_STAGE = getCurrentStage(__filename);
+
+const func: DeployFunction = trackFinishedStage(CURRENT_STAGE, async function (hre: HRE) {
+  const { deploy, deployer } = await prepareDeployment(hre);
+
+  const wethAddress = await findDependency(WETH);
+  const controllerAddress = await findDependency(CONTROLLER);
+  const zeroExExchangeAddress = await findDependency(ZERO_EX_EXCHANGE);
+
+  await deployExchangeIssuanceZeroEx(
+    CONTRACT_NAMES.EXCHANGE_ISSUANCE_ZEROEX,
+    wethAddress,
+    controllerAddress,
+    zeroExExchangeAddress,
+  );
+
+  async function deployExchangeIssuanceZeroEx(
```

Comment:
Minor nit, but if this is just an async function we wait for, why not just drop `async`, and call the function normally?

Reply:
Yes, agree that definitely makes sense here.